youtube_dl/extractor/neteasemusic.py

```diff
@@ -108,27 +106,44 @@ class NetEaseMusicIE(NetEaseMusicBaseIE):
             'ext': 'mp3',
             'title': '소원을 말해봐 (Genie)',
             'creator': '少女时代',
-            'description': 'md5:79d99cc560e4ca97e0c4d86800ee4184',
             'upload_date': '20100127',
             'timestamp': 1264608000,
             'alt_title': '说出愿望吧(Genie)',
         }
     }]
 
+    def _fix_timestamp(self, timestamp):
+        # Netease returns timestamp use 2 or 3 digits for less than
+        # 1 second metrics
+        # While standard LRC requires exact 2 digits
+        # remvoe the 3rd digit if the there're three
+        match_expr = r'\[([0-9]{2}:[0-9]{2}\.[0-9]{2})(.*)\]'
+        match_result = re.match(match_expr, timestamp)
+        if match_result:
+            # This must match as match_expr is exact the first group of lyrics_expr
+            return "[" + match_result.group(1) + "]"
+        else:
+            # return a valid timestamp to avoid other applications' error
+            return "[00:00.00]"
+
     def _process_lyrics(self, lyrics_info):
         original = lyrics_info.get('lrc', {}).get('lyric')
         translated = lyrics_info.get('tlyric', {}).get('lyric')
 
         if not translated:
-            return original
+            translated = ""
+        if not original:
+            original = ""
 
-        lyrics_expr = r'(\[[0-9]{2}:[0-9]{2}\.[0-9]{2,}\])([^\n]+)'
+        lyrics_expr = r'(\[[0-9]{2}:[0-9]{2}\.[0-9]{2,}\])([^\n]*)'
+        lyrics_expr_multitimetag = r'(\[[0-9]{2}:[0-9]{2}\.[0-9]{2,}\])+([^\n]*)'
         original_ts_texts = re.findall(lyrics_expr, original)
         translation_ts_dict = dict(
-            (time_stamp, text) for time_stamp, text in re.findall(lyrics_expr, translated)
+            (time_stamp, text) for time_stamp, text in re.findall(lyrics_expr_multitimetag, translated)
         )
-        lyrics = '\n'.join([
-            '%s%s / %s' % (time_stamp, text, translation_ts_dict.get(time_stamp, ''))
+        lyrics = ''.join([
+            '%s%s / %s\n' % (self._fix_timestamp(time_stamp), text, translation_ts_dict.get(time_stamp)) if translation_ts_dict.get(time_stamp, None)
```

Comment:
Combining is a task of a player or any other consumer, youtube-dl should not do that. Otherwise we would need to generate all `n! / (n - 2)!` partial permutations of unique subtitles for consistency.